src/UserInterfaceServiceProvider.php

```diff
@@ -175,7 +175,7 @@ private function registerBladeComponents(): void
         Blade::component('ark::inputs.date-picker', 'ark-date-picker');
         Blade::component('ark::inputs.input', 'ark-input');
         Blade::component('ark::inputs.input-with-icon', 'ark-input-with-icon');
-        Blade::component('ark::inputs.input-with-prefix', 'ark-input-with-prefix');
+        Blade::component('ark::inputs.input-with-control', 'ark-input-with-control');
```

Comment:
can this be changed to two components:
- `input-with-prefix`
- `input-with-suffix`

you can then have those two components as wrappers that both call the `input-with-control` component but with a predefined property for the location of it. Reason being is that:
1. it avoids a breaking change as the existing prefix component will still exist
2. it makes it clearer (to me at least) what the component does, as `input-with-control` feels ambiguous to me (what's a control?)

Reply:
@ItsANameToo done